Wikipedia/UI-V5/WMFArticleListCollectionViewController.m

```diff
@@ -54,6 +55,7 @@ - (instancetype)initWithCoder:(NSCoder*)aDecoder {
 }
 
 - (void)commonInit {
+    [self observeArticleUpdates];
```

Comment:
doing anything that touches views in `init` of a view controller is ill-advised. in this case, an article download elsewhere could have the unintended side effect of forcing the view to load prematurely.  we should at least wait until `viewDidLoad`. IMHO it would be more efficient to observe (and update) on `viewWillAppear` and un-observe in `viewDidDisappear`.  This means views which aren't shown are inert, and will **lazily** update when they're shown again.